functional_tests/scylla_operator/test_functional.py

```diff
@@ -208,16 +226,14 @@ def get_pod_names_and_ips(cluster_name: str, namespace: str):
             need_to_collect_logs = False
         finally:
             if need_to_collect_logs:
-                KubernetesOps.gather_k8s_logs(
-                    logdir_path=logdir, kubectl=kubectl, namespaces=[namespace, namespace2])
+                KubernetesOps.gather_k8s_logs(logdir_path=logdir, kubectl=kubectl, namespaces=[namespace, namespace2])
```

Comment:
The same here.
It was on 2 lines because of the length.

Reply:
line length is now 112